### ISSUE TITLE
Frustrated at lack of documentation...can anyone help?

### ISSUE THREAD

**Issue by TheNewbFlyer** [PowerBroker2/ArduPID#9]
The variable "Setpoint" is the desired angle correct? What is its min/max values (the example just has "512")?

I normally use VS Code, but I went to the Arduino IDE to use the Serial Plotter to diagnose things, but it's just a bunch of lines flying by really fast. How can I display the values correctly (I copied this part straight from the example).

**Comment by alexiscruzdavid** [comment]
Have you tried adding a delay so the lines come by slower? That's what I typically do when debugging. I do agree with you on the lack of documentation

**Comment by PowerBroker2** [comment]
I'm busy with other projects atm, but will eventually take a look at this lib again. As for now, there is an example (which you've found), but there is also a readme that explains quite a few things. The setpoint and gain parameters are interfaced in the same exact way as the legacy Arduino PID library - pass the parameters by reference to the `begin()` function and change the parameters directly when needed without being required to call any lib functions.

Also, the purpose of the setpoint is simply the desired controller output. That desired output will change depending on your application.

**Comment by PowerBroker2** [comment]
Yeah, absolutely! It would be super cool if it was compatible with Read the Docs. I don't have experience with Read the Docs, but I've tried messing around with doxygen (had issues with getting it to work and pretty much abandoned it).

I'm open to PRs/MRs
